posthog/tasks/usage_report.py

```diff
@@ -500,15 +504,23 @@ def get_teams_with_event_count_by_name(begin: datetime, end: datetime) -> List[T
 
 @timed_log()
 @retry(tries=QUERY_RETRIES, delay=QUERY_RETRY_DELAY, backoff=QUERY_RETRY_BACKOFF)
-def get_teams_with_recording_count_in_period(begin: datetime, end: datetime) -> List[Tuple[int, int]]:
+def get_teams_with_recording_count_in_period(
+    begin: datetime, end: datetime, snapshot_source: Literal["mobile", "web"]
+) -> List[Tuple[int, int]]:
     previous_begin = begin - (end - begin)
 
     result = sync_execute(
         """
         SELECT team_id, count(distinct session_id) as count
-        FROM session_replay_events
-        WHERE min_first_timestamp BETWEEN %(begin)s AND %(end)s
-        AND session_id NOT IN (
+        FROM (
+            SELECT any(team_id) as team_id, session_id
+            FROM session_replay_events
+            WHERE min_first_timestamp BETWEEN %(begin)s AND %(end)s
+            GROUP BY session_id
+            -- exclude mobile recordings here
```

Comment:
why do we exclude mobile recordings here? shouldn't the `where` with `snapshot_source` include only actually what you want? either mobile or web.

Reply:
ah, rushing to get the PR In before doing the school run.

Yep, the comment is out of sync with reality...

Reply:
```suggestion
```